FrontDesk/tests/UnitTests/Core/AggregatesEntities/ScheduleTests/Schedule_Handle.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Threading.Tasks;
+using AutoFixture;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.ScheduleTests
+{
+  public class Schedule_Handle
+  {
+    private Fixture _fixture = new Fixture();
+
+    [Fact]
+    public async Task SetsProperties()
+    {
+      // verify this method
+      //     public Schedule(Guid id, DateTimeRange dateRange, int clinicId, IEnumerable<Appointment> appointments)
+
```

Comment:
These are the tests I was hoping you'd write. Perhaps in the next PR?

Reply:
ok